ballerina/tests/10-custom-datetime-test.bal

```diff
@@ -34,26 +36,28 @@ isolated function beforeInsertTimeFunc() returns sql:Error? {
        PRIMARY KEY(PK)
        )`
    );
-
    check oracledbClient.close();
 }
 
 @test:Config {
-   groups:["execute"]
+   groups:["datetime"]
 }
 isolated function insertIntervalWithString() returns sql:Error? {
    Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT);
    string date = "05-JAN-2020 10:35:10 AM";
+   string dateOnly = "05-JAN-2020";
+   string timeOnly = "0 11:00:00";
```

Comment:
Should the time only format be hh:mm:ss? Without the leading 0 for the day.

Reply:
Intervals will handle separately in issue https://github.com/ballerina-platform/ballerina-standard-library/issues/1763